assets/js/romo/select.js

```diff
@@ -10,44 +10,40 @@ var RomoSelect = function(element) {
 
   this.doSetValue(this._elemValues());
 
-  if (this.elem.attr('id') !== undefined) {
-    $('label[for="'+this.elem.attr('id')+'"]').on('click', $.proxy(function(e) {
+  if (Romo.attr(this.elem, 'id') !== undefined) {
+    var labelElem = Romo.f('label[for="'+Romo.attr(this.elem, 'id')+'"]')[0]
+    labelElem.on('click', Romo.proxy(function(e) {
       this.romoSelectDropdown.doFocus();
     }, this));
   }
 
-  $(window).on("pageshow", $.proxy(function(e) {
+  Romo.on(window, "pageshow", Romo.proxy(function(e) {
     this._refreshUI();
   }, this));
 
-  this.elem.on('select:triggerSetValue', $.proxy(function(e, value) {
+  Romo.on(this.elem, 'romoSelect:triggerSetValue', Romo.proxy(function(e, value) {
     this.doSetValue(value)
   }, this));
 
-  this.elem.trigger('select:ready', [this]);
+  Romo.trigger(this.elem, 'romoSelect:ready', [this]);
 }
 
 RomoSelect.prototype.doInit = function() {
   // override as needed
 }
 
 RomoSelect.prototype.doSetValue = function(value) {
-  var values = undefined;
-  if (Array.isArray(value)) {
-    values = value;
-  } else {
-    values = [value];
-  }
-  values = values.filter($.proxy(function(v) {
-    return this.elem.find('OPTION[value="'+v+'"]')[0] !== undefined;
+  var values = Romo.array(value);
```

Comment:
@jcredding note: this doesn't work for single values (which is what the old implementation was trying to handle).  We need to handle making `.array` work for single values in a sep effort to keep this implementation (which I'd like to do).